target/linux/mediatek/mt7622/base-files/etc/hotplug.d/ieee80211/11_fix_wifi_mac

```diff
@@ -0,0 +1,18 @@
+[ "$ACTION" == "add" ] || exit 0
+
+PHYNBR=${DEVPATH##*/phy}
+
+[ -n $PHYNBR ] || exit 0
+
+. /lib/functions.sh
+. /lib/functions/system.sh
+
+board=$(board_name)
+
+case "$board" in
+	ruijie,rg-ew3200gx-pro)
+		hw_mac_addr=$(mtd_get_mac_ascii u-boot-env ethaddr)
```

Comment:
No need to read the address from flash now that it gets passed down to Linux by U-Boot.

Reply:
`[ "$PHYNBR" = "0" ] && macaddr_add $(get_mac_label) 3 > /sys${DEVPATH}/macaddress`
`[ "$PHYNBR" = "1" ] && macaddr_add $(get_mac_label) 2 > /sys${DEVPATH}/macaddress`
Is it right?

Reply:
Yes, that looks good.